src/main/java/org/betonquest/betonquest/database/Connector.java

```diff
@@ -69,7 +69,11 @@ public ResultSet querySQL(final QueryType type, final String... args) {
         try {
             final PreparedStatement statement = connection.prepareStatement(sql);
             for (int i = 0; i < args.length; i++) {
-                statement.setString(i + 1, args[i]);
+                try {
+                    statement.setInt(i + 1, Integer.parseInt(args[i]));
+                } catch (NumberFormatException nfe) {
+                    statement.setString(i + 1, args[i]);
+                }
```

Comment:
i did not found any reason for this change. I suggest to change it back, otherwise you need to expain this change to me.

Reply:
The limit must be set as an integer value for some databases such as MariaDB. When I tested it with the setString method I got an error.

Do you want me to put it in a comment in the code too?

Reply:
I created a PR in your repository https://github.com/Archerymaister/BetonQuest/pull/1 fixing this issue
may @SaltyAimbOtter will review this first, before you merge it, but i am sure you can also merge it and salty will review it afterwards.

Reply:
hey sorry I wasn't responding, I'm currently on vacation and nowhere near my computer! is there anything left to do for me? i am back on Wednesday

Reply:
nothing left to do for you ;)